packages/requester-utils/src/BaseResource.ts

```diff
@@ -36,6 +37,47 @@ function getDynamicToken(tokenArgument: (() => Promise<string>) | string): Promi
   return tokenArgument instanceof Function ? tokenArgument() : Promise.resolve(tokenArgument);
 }
 
+const DEFAULT_RATE_LIMITS = Object.freeze({
+  // Default rate limit
```

Comment:
I know I could look it up, but it would be great to know the time arguments on these as part of the comment. Is it per second?

Reply:
I had to change it so its per minute, since i believe thats how the docs describe it.

Reply:
Per *minute*...wow, that's pretty limiting. Definitely need this patch in then!

Reply:
I could change the default to 300 maybe? I tried finding to see if there was a default rate limit but couldnt see anything specific in the docs. The rest though are taken directly from the docs

Reply:
Though this older doc says more like 1000: https://about.gitlab.com/blog/2020/12/17/automation-check-in-and-rate-limit-changes-on-gitlab-dot-com/

Reply:
1000/minute seems like a reasonable generic. We won't really know until it gets thrown at the wall and it's both an easy override and an easy fix.